pkg/internal/testing/integration/etcd.go

```diff
@@ -86,7 +86,7 @@ func (e *Etcd) setProcessState() error {
 		return err
 	}
 
-	e.processState.StartMessage = internal.GetEtcdStartMessage(e.processState.URL)
+	e.processState.HealthCheckEndpoint = "/health"
```

Comment:
o_O

Reply:
`StartMessage` is deprecated. So staticcheck is failing. I was iffy on this change too, we can add `//nolint` to revert this change

https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/internal/testing/integration/internal/process.go#L40

Reply:
Before it was `internal.GetEtcdStartMessage(e.processState.URL)`, however,, now it is receiving a fixed value. Is it accurate? Could not it cause issues?